### PR TITLE
[#298] handle message and error on server side

### PR DESCRIPTION
**Pull request by modosc**

### CODE REVIEW

File: src/server/transforms.js [hunk 1]
```diff
@@ -71,6 +71,9 @@ function buildErrorData(item, options, callback) {
   item.data.body = item.data.body || {};
   item.data.body.trace_chain = chain;
 
+  // clear out any existing message
+  delete item.data.body.message
```
Comment (coryvirok) [inline src/server/transforms.js]: I'd like to not lose this information. I think the preferred way to handle this is to implement what the browser transforms do:

From https://github.com/rollbar/rollbar.js/blob/master/src/browser/transforms.js#L99
```js
function addBody(item, options, callback) {
  if (item.stackInfo) {
    addBodyTrace(item, options, callback);
  } else {
    addBodyMessage(item, options, callback);
  }
}
```

I would refactor the server-side to remove the `addMessageData()` transform and add the `addBody()` transform in its place. That code *may* need to be tweaked to remove the item.message in the case we have an error as well. In this case, we should set the `error.description = message`. @rokob can provide more details.

**Comment by coryvirok** [comment]
Thanks for this PR! I think there's a more preferred way to handle this case. See my comment.

**Comment by modosc** [comment]
thans @coryvirok - that's way more detailed of a change than i have time to handle right now. hopefully someone else can pick this up asap?

**Comment by coryvirok** [comment]
No prob. We will implement the fix once @rokob is back in town on Monday.

**Comment by sarkistlt** [comment]
Hi, the same issue, is it already fixed?

**Comment by fijimunkii** [comment]
Hi I am experiencing this same issue. Any progress?

**Pull request closed without merging**